### DISCUSSION TITLE
Preparation for initial 1.0.0 release

### DISCUSSION THREAD

**Discussion by mnonnenmacher** [eclipse-apoapsis/ort-server#884] [General]
I have prepared a milestone for the initial 1.0.0 release of the ORT Server:
https://github.com/eclipse-apoapsis/ort-server/issues?q=is%3Aissue+milestone%3A1.0.0+

As we are already running the server in production for several months, I do not see any new features as requirements for the initial release. Also, there are currently no critical bugs. Therefore I have only added issues related to technical or governance topics to the milestone. If you think any issues should be added or removed, please let me know.

Please note that we are planning to do frequent releases with improvements and bug fixes, probably on a weekly basis.

**Comment by sschuberth** [comment]
What about issues #100 and #123?

**Reply by mnonnenmacher** [reply]
That would delay the initial release by several weeks at least. We are still clarifying what our internal requirements are for keeping user data and will have to align the server implementation with that. Once we have that we will have to align with you if that matches your requirements.

The general solution will be like recently discussed, that there are different hard- and soft-deletes. However, what exactly soft-delete does needs to be specified (some data will have to be kept for statistical, legal, or billing requirements).

**Comment by mnonnenmacher** [comment]
I have created a new 0.1.0 milestone for the first release, we will start by making release candidates for 0.1.0. The release candidates will be used to make sure that the release automation works and as input for the EF release review.

Features that we see as mandatory for a 1.0.0 release should be added to the 1.0.0 milestone, I have already added #100 and #123 which were mentioned by @sschuberth.

**Reply by sschuberth** [reply]
Can we also get https://github.com/eclipse-apoapsis/ort-server/issues/193 for 1.0.0?

**Reply by mnonnenmacher** [reply]
I have added this and a few more (also already closed ones for reference). We need to be careful that we don't make the scope too large, but we can always adapt.
